data/gamerules.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# A set of constants and booleans that determine game behavior.
+gamerules
+	# Determines whether the "universal ramscoop" is active on all ships. This ramscoop
+	# provides a very small amount of fuel to every ship regardless of whether they have
+	# an actual ramscoop or not. The strength of this ramscoop is scaled with the distance
+	# to the system center twice, meaning it falls off much more quickly than a normal ramscoop.
+	"universal ramscoop" true
+	
+	# A person ship spawns on average every this many frames. The default is once every ten minutes.
+	"person spawnrate" 36000
```

Comment:
should it be `spawn rate`?

Reply:
Probably, but it's one of those pairs of words to me that just seems like eventually it'll become one word, like how web site became website. There's a few other such pairs of words that I've been badgered about sometimes pressing together, like eachother or everytime. (Like, come on, at least with everytime, everyone and everywhere are already accepted single words.) Gamerule should also technically be "game rule," although that one being one word is literally just because the command in Minecraft is "gamerule."